src/aics-image-viewer/components/AxisClipSliders/styles.css

```diff
@@ -72,8 +80,22 @@
     max-width: 85px;
   }
 
+  .slider-play-buttons {
+    margin-top: 2px;
+  }
+
   &.clip-sliders-2d {
-    max-width: 780px;
+    /* Make room for play buttons */
+    max-width: 755px;
+
+    /* Slider slice display contains 2 numbers in 2d mode, not 3 - shrink accordingly */
+    .slider-slices {
+      flex: 0 0 60px;
+    }
+
+    .axis-slider-container {
+      flex: 0 1 592px;
```

Comment:
in here I see 592, 755, and 617px specified.  Are these magic numbers coming directly from the design document?

Reply:
In order to get flexbox to scale, center, and wrap things right, I had to specify the widths of several elements (e.g. slice numbers) beyond what was in the design document. (We would have gotten bad results from unspecified widths anyways, e.g. the centered position of a slider being partially dependent on the width of the slice numbers, and therefore on the position of the slider handle.) These large magic numbers are container elements that also need a baseline width, and are derived from the sum of their children's widths. (This probably should at least be documented.)

Reply:
the comments really help a lot